<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vis4xe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="statesStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>960</width>
      <height>540</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>960</width>
      <height>540</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="homePage">
     <widget class="QLabel" name="projectNameLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>960</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 44px;
background-color: #0099ff;
color: #333333;
font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;</string>
      </property>
      <property name="text">
       <string>HomePage</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QStackedWidget" name="projectsExists">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>900</width>
        <height>270</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="projectsList"/>
      <widget class="QWidget" name="noProjectsInfo">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>120</y>
          <width>791</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #FFFFFF;
font-size: 20px;</string>
        </property>
        <property name="text">
         <string>You currently do not have any projects. To create a new project press the button below</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0099ff;
font-size:20px;
border-radius: 25px;</string>
      </property>
      <property name="text">
       <string>Vis4xe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="addNewButton">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>450</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#addNewButton
{
	border-radius:  125px;
	border: 3px solid #00b300;
	color: #00b300;
	font-size: 56px;
	text-align:top;
	padding-top:20px;

font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;
}

#addNewButton:hover
{
	background-color: #00b300;
	color:#333333;
}</string>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openProjectButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>35</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#openProjectButton
{
	border-radius:  25px;
	color: #0099ff;
	font-size: 26px;
	font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;
}

#openProjectButton:hover
{
	border:3px solid #333333;
	background-color: #0099ff;
	color:#333333;
}</string>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addPage">
     <widget class="QLabel" name="addingProjectHeader">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>960</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 44px;
background-color: #0099ff;
color: #333333;
font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;</string>
      </property>
      <property name="text">
       <string>Adding Project</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
font-size: 20px;
background-color: #0099ff;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="colsLineEdit">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lcdSizeLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>230</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
font-size: 20px;
background-color: #0099ff</string>
      </property>
      <property name="text">
       <string>Lcd size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="colsLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
font-size: 15px;
background-color: #0099ff</string>
      </property>
      <property name="text">
       <string>Cols</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="rowsLineEdit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rowsLabel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>280</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;
font-size: 15px;
background-color: #0099ff</string>
      </property>
      <property name="text">
       <string>Rows</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="finalAddButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>390</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#finalAddButton
{
	border-radius:  60px;
	border: 3px solid #00b300;
	color: #00b300;
	font-size: 44px;
	font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;
}

#finalAddButton:hover
{
	background-color: #00b300;
	color:#333333;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>415</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;1, 24&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>&lt;1, 10&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_alert">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
font-size: 16px;
</string>
      </property>
      <property name="text">
       <string>You have to type project name</string>
      </property>
     </widget>
     <widget class="QLabel" name="cols_alert">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
font-size: 16px;</string>
      </property>
      <property name="text">
       <string>Cols must be between 1 and 24</string>
      </property>
     </widget>
     <widget class="QLabel" name="rows_alert">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>320</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;
font-size: 16px;</string>
      </property>
      <property name="text">
       <string>Rows must be between 1 and 10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #0099ff;
font-size:20px;
border-radius: 25px;</string>
      </property>
      <property name="text">
       <string>Vis4xe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelAddingButton">
      <property name="geometry">
       <rect>
        <x>885</x>
        <y>465</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#cancelAddingButton
{
	border-radius:  75px;
	border: 3px solid red;
	color: red;
	font-size: 26px;
	text-align:left;
    text-align:top;	
	padding-top: 30px;
    padding-left:-65px;

font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;
}

#cancelAddingButton:hover
{
	background-color: red;
	color:#333333;
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="editPage">
     <widget class="QLabel" name="stringsHeader">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>0</y>
        <width>190</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333333;
font-size: 36px;
background-color: #0099ff;
font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;</string>
      </property>
      <property name="text">
       <string>Strings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="editingQuitButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#editingQuitButton
{
	border: 2px solid red;
	color: red;
	border-radius:30px;
	font-size:20px;
}


#editingQuitButton:hover
{
	background-color: red;
	color: #333333;
}
</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStringButton">
      <property name="geometry">
       <rect>
        <x>585</x>
        <y>470</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#addStringButton
{
	border: 2px solid #0099ff;
	color: #0099ff;
	border-radius:30px;
	font-size:20px;
}

#addStringButton:hover
{
	background-color: #0099ff;
	color: #333333;
}
</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stringValueLine">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>485</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #333333;
color: #ffffff;
border: 2px solid #0099ff;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>28</number>
      </property>
     </widget>
     <widget class="QPushButton" name="editingQuitWithSaveButton">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>470</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#editingQuitWithSaveButton
{
	border: 2px solid #00cc00;
	color: #00cc00;
	border-radius:30;
	font-size: 20px
}


#editingQuitWithSaveButton:hover
{
	background-color: #00cc00;
	color: #333333;
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed,Extra 12</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 3px solid #0099ff;
border-bottom: 3px solid #0099ff;
border-left: 3px solid #0099ff;

color: #FFFFFF;

font-family: &quot;Tw Cen MT Condensed&quot;,  Extra Bold;
font-size:44px;
</string>
      </property>
      <property name="text">
       <string>Editing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333333;
font-size:20px;
border-radius: 20px;
background-color:#0099ff</string>
      </property>
      <property name="text">
       <string>Vis4xe</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>stringsHeader</zorder>
     <zorder>label</zorder>
     <zorder>label_7</zorder>
     <zorder>editingQuitWithSaveButton</zorder>
     <zorder>editingQuitButton</zorder>
     <zorder>stringValueLine</zorder>
     <zorder>addStringButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
